examples/nextjs-with-typescript-v4-migration/package.json

```
@@ -17,7 +17,10 @@
     "@mui/icons-material": "latest",
     "@mui/material": "latest",
     "@mui/styles": "latest",
+    "autoprefixer": "^10.4.7",
+    "clean-css": "^5.3.0",
     "next": "12.1.5",
```

I think that it would be healthier to use "latest" for all the dependencies in the examples, helping make sure we keep them up to date.

```suggestion
    "next": "latest",
```

---

We need to lock some of the dependencies in this example, because @mui/material does not work with React 18. +1 for using latest of next, but this does not apply on all dependencies. We can however use latest on all of the dependencies added in this PR. cc @paustria